src/pymorize/std_lib/__init__.py

```diff
@@ -184,14 +189,24 @@ def temporal_resample(
     - 'D': daily
     - 'H': hourly
 
-    The resampling method (mean, sum, etc.) is determined by the rule.
+    See Also
+    --------
+    https://docs.xarray.dev/en/stable/user-guide/time-series.html#resampling-and-grouped-operations
     """
     if not has_time_axis(data):
         return data
 
     time_dim = get_time_label(data)
     freq = rule.data_request_variable.frequency
-    return data.resample({time_dim: freq}).mean()
+    if not freq_is_coarser_than_data(freq, data):
+        raise PymorizeResamplingTimeAxisIncompatibilityError(
+            f"Requested frequency {freq} is finer than the dataset's inherent frequency. Cannot resample!"
```

Comment:
Can you add to the error a mention about which model variable is affected?

Reply:
Sure, here is something:

```suggestion
            f"Requested frequency {freq} for cmor variable {rule.cmor_variable} is finer than the dataset's ({rule.model_variable}) inherent frequency. Cannot resample!"
```